jax/_src/api.py

```diff
@@ -3131,11 +3131,7 @@ def clear_backends():
   xb._clear_backends()
   xb.local_devices.cache_clear()
   xb.process_count.cache_clear()
-  dispatch.xla_primitive_callable.cache_clear()
   util.clear_all_caches()
-  pjit._infer_params_cached.cache_clear()
```

Comment:
Why remove this? You are not registering these caches in the register_cache function too.

I don't think we should remove these cache clears.

Reply:
Of course